mobile-app/app/api/wallet/notifications.ts

```diff
@@ -0,0 +1,77 @@
+import * as Notifications from 'expo-notifications'
+import { Platform } from 'react-native'
+import { PermissionStatus } from 'expo-modules-core'
+
+export interface SendNotificationProps {
```

Comment:
```suggestion
export interface SendNotificationData {
```

avoid use of Props for non React common to reduce confusion

Reply:
Will do the needful